### PR TITLE
docs: remove unrelated doc

### PR DESCRIPTION
**Pull request by Roiocam**
We are the community, which means we must support them.

### CODE REVIEW

File: docs/src/main/paradox/discovery/marathon.md [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 @@@ warning
 
-This module is community maintained and the Lightbend subscription doesn't cover support for this module.
-  It is also marked as @extref:[may change](pekko:common/may-change.html).
-  That means that the API, configuration or semantics can change without warning or deprecation period.
+It is marked as @extref:[may change](pekko:common/may-change.html).
```
Comment (pjfanning) [inline docs/src/main/paradox/discovery/marathon.md:5]: Thanks but could you change this to `This module is marked ...`

File: docs/src/main/paradox/discovery/consul.md [hunk 2]
```diff
@@ -2,9 +2,8 @@
 
 @@@ warning
 
-This module is community maintained and the Lightbend subscription doesn't cover support for this module.
-  It is also marked as @extref:[may change](pekko:common/may-change.html).
-  That means that the API, configuration or semantics can change without warning or deprecation period.
+It is marked as @extref:[may change](pekko:common/may-change.html).
```
Comment (pjfanning) [inline docs/src/main/paradox/discovery/consul.md:5]: Could you change this to `This module is marked ...` ?

File: docs/src/main/paradox/discovery/marathon.md (hunk 1 above)
Comment (Roiocam) [reply docs/src/main/paradox/discovery/marathon.md:5]: I will fix this after sometime, i have to build the document and verify first, and then i will fix this PR.

File: docs/src/main/paradox/discovery/consul.md (hunk 2 above)
Comment (Roiocam) [reply docs/src/main/paradox/discovery/consul.md:5]: Nice input, thanks.

**Pull request closed without merging**